src/main/java/it/pagopa/pn/deliverypush/action2/utils/InstantNowSupplier.java

```diff
@@ -0,0 +1,13 @@
+package it.pagopa.pn.deliverypush.action2.utils;
+
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+
+@Component
+public class InstantNowSupplier {
```

Comment:
Sarebbe meglio fare un interfaccia e un implementazione, che nei test sarà differente

Reply:
spostiamo in ISSUE